README.md

```diff
@@ -1,69 +1,45 @@
-!["The Evolution of Trust"](https://i.imgur.com/kde760y.png)
+!["Ewolucja Zaufania"](https://sin.github.io/trust/assets/logo.png)
 
-#	PLAY IT HERE: [http://ncase.me/trust/](http://ncase.me/trust/)
+# ZAGRAJ TUTAJ: [sin.github.io/trust](https://sin.github.io/trust/)
 
-*The Evolution of Trust* is dedicated to the public domain, and is made thanks to the many Creative Commons & open source resources out there! Here is a full list of music, sounds, and code used:
+Gra o tym dlaczego i w jaki sposób sobie ufamy, przetłumaczona na język polski.
```

Comment:
Może lepiej:
Gra o tym w jaki sposób budujemy zaufanie i co z tego wynika. Polskie tłumaczenie.
Gra o tym jak zaufanie zmienia świat. Polskie tłumaczenie.

Jeśli żadna z wersji Wam nie odpowiada, to chociaż zmieńmy: sobie ufamy na budujemy zaufanie.